src/components/card/fixed.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC, ReactNode } from "react"
+
+interface Props {
+  renderImage: () => ReactNode
+  children: ReactNode
```

Comment:
I think you don't need to type children in the props of a FC as they are provided by the FC interface

Reply:
i will check, thanks for letting me know

Reply:
i see that all the other components have the children as a prop...so i thought we go with the same pattern. WDYT?

Reply:
No strong oppion its just not needed up to you

Reply:
We do require children here, whereas they are optional in FC type. I'd keep it.